### ISSUE TITLE
Target - web service/API/message call

### ISSUE THREAD

**Issue by prybak** [pluginsGLPI/formcreator#2303]
**Is your feature request related to a problem? Please describe.**
Currently From Creator allows to create change or ticket or mixture of them. It would be great  to have an option to call some web service/API/send message to some message-broker service. It highly will improve automation of handling tickets.

**Describe the solution you'd like**
New target:
- API Call that allows to configure that call and pass data from collected form.
- Message that allows to configure and send message to some message-broker.

Feature should be able to perform multiple actions (Form Creator should be able to add ticket, change as well as call some API/send message).

**Describe alternatives you've considered**
Use some external tools to scan for tickets and call desired actions.

**Additional context**
Automation, automation, automation.

**Comment by btry** [comment]
Hi

It appears that you need an API callback or a client of a message queue protocol.

**Comment by prybak** [comment]
Yes something like that. Currently we are creating tickets with some specific titles/tags and we have some Powershell script that looks for those tickets and executes appropriate actions. This approach is fine when we are talking about a few actions but not if we would like to have many of them.

**Comment by prybak** [comment]
You can make us of this plugin ;)
https://github.com/FutureProcessing/glpi-webhook

**Comment by glasserre** [comment]
Very interesting by this impementation, the idea is to integrate GLPI with existing systems.


> You can make us of this plugin ;) https://github.com/FutureProcessing/glpi-webhook

Not really a solution, we need to call an api and pass questions answers in body request. 
For example : Onboarding, HR fill basic new employee information. On submit, call an API (Azure Logic App or runbook webhook) that will create the account (AD, mail, IM, ....), grant permissions on 3rd party systems, etc ...

Will really love to use this.

**Comment by btry** [comment]
Hi

If you have a valid subscription to [GLPI Network](https://services.glpi-network.com/)  you may request for a quote.
